app/ts/components/TokenList.tsx

```diff
@@ -0,0 +1,36 @@
+import { TokenMetaData } from '../library/tokens.js'
+import { removeNonStringsAndTrim } from '../library/utilities.js'
+import { TokenIcon } from './Icon/Tokens/index.js'
+
+export const TokenList = () => {
+	// const active = useSignal<string | undefined>(undefined)
+	// let children = []
+	// for (const [id, metadata] of tokenList) {
+	// 	children.push(<TokenListItem key={id} isActive={active.value === id} metadata={metadata} onChange={address => (active.value = address)} />)
+	// }
+	// return <div class='bg-black text-white'>{children}</div>
```

Comment:
Is this commented out intentionally?

Reply:
Yes, I separated the listing changes on another PR where the new design is also applied #75